app/services/exports/export_distributions_csv_service.rb

```diff
@@ -1,10 +1,14 @@
 module Exports
   class ExportDistributionsCSVService
-    def initialize(distribution_ids:)
-      # Use a where lookup so that I can eager load all the resources needed
-      # rather than depending on external code to do it for me. This makes
-      # this code more self contained and efficient!
-      @distributions = Distribution.includes(:partner, :storage_location, line_items: [:item]).where(id: distribution_ids).order('issued_at DESC')
```

Comment:
@armahillo do we need to eager load any of these that the controller isn't already doing?

Reply:
It looks like they _might_ be missing in the two actions -- I can add them so they're explicitly included.

Reply:
Done! It actually addressed a Bullet notification on `distributions#index` 

tested both pages manually and they load ok :+1: